wagtail/admin/widgets/button.py

```diff
@@ -148,7 +148,7 @@ def dropdown_buttons(self):
 
     def get_context_data(self, parent_context):
         return {
-            "buttons": self.dropdown_buttons,
+            "buttons": sorted(self.dropdown_buttons),
```

Comment:
Moving the sorting here so that it's also applied to both `ButtonWithDropdown` and `ButtonWithDropdownFromHook`, plus also ensure it's sorted even if `dropdown_buttons` is overridden.